src/css/hero.css

```diff
@@ -0,0 +1,110 @@
+.hero {
+  position: relative;
+  height: 709px;
+  padding-bottom: 72px;
+  max-width: 335px;
+  display: flex;
+  flex-direction: column;
+  gap: 40px;
+  justify-content: flex-end;
+  margin: 0 auto;
+
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 16px;
+  background:
+    linear-gradient(357deg, rgba(0, 0, 0, 0.4) 39.23%, rgba(0, 0, 0, 0) 66.28%),
+    url(<path-to-image>),
```

Comment:
specify path the image
add image to the `img` folder
add images for different resolutions and dpr